src/librustc_mir/borrow_check/nll/constraints/graph.rs

```diff
@@ -42,7 +42,7 @@ impl ConstraintGraph {
     }
 
     /// Given a region `R`, iterate over all constraints `R: R1`.
-    crate fn outgoing_edges(&self, region_sup: RegionVid) -> Edges<'_> {
+    crate fn outgoing_edges(&'self_ self, region_sup: RegionVid) -> Edges<'self_> {
```

Comment:
Why are you using a named lifetime here? I would've thought the original signature would have worked fine

Reply:
Looks like that's the case here -- I think this might've been before I properly understood what `'_` meant.